src/test/java/com/example/cats/integration/WireMockTest.java

```diff
@@ -0,0 +1,35 @@
+package com.example.cats.integration;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.junit.jupiter.api.*;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+class WireMockTest {
```

Comment:
It's awesome that you tried sth new - but this is not testing anything except that you can run the wiremock, please rewatch  - https://youtu.be/xcw5veP0TPw?si=kcvFz2Zr6lqcNsKL&t=2262